forge-game/src/main/java/forge/game/ability/effects/TokenEffectBase.java

```diff
@@ -112,7 +112,8 @@ protected TokenCreateTable makeTokenTable(TokenCreateTable tokenTable, final boo
             Card prototype = c.getColumnKey();
             Player creator = c.getRowKey();
             Player controller = prototype.getController();
-            int cellAmount = c.getValue();
+            int cellAmount = !sa.hasParam("DifferentAmounts") ? c.getValue() :
```

Comment:
this seems weird, wouldn't that mess up replacement effects?
any reason not to just use ChangeZoneTable$?

Reply:
That is a better idea